traffic_monitor/manager/monitorconfig.go

```diff
@@ -146,11 +148,12 @@ func StartMonitorConfigManager(
 		staticAppData,
 		toSession,
 		toData,
+		url6Regex,
 	)
 	return monitorConfig
 }
 
-const DefaultHealthConnectionTimeout = time.Second * 2
+const DefaultHealthConnectionTimeout = time.Second * 20
```

Comment:
What's the thinking behind this? That's a pretty big jump

Reply:
in testing it seemed necessary but must have been an environment issue because now 2 seems fine. reverted